VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.test.tsx

```diff
@@ -71,14 +75,31 @@ context('ClaimDetailsScreen', () => {
             ...claimData,
           },
         })
-      renderWithData({
+      renderWithData(false, {
         ...claimData,
       })
       await waitFor(() => fireEvent.press(screen.getByText('Details')))
       await waitFor(() => fireEvent.press(screen.getByText('Details')))
 
       await waitFor(() => expect(screen.getByText('Claim type')).toBeTruthy())
     })
+
+    it('should display the Files component', async () => {
+      when(api.get as jest.Mock)
+        .calledWith(`/v0/claim/0`, {}, expect.anything())
+        .mockResolvedValue({
+          data: {
+            ...claimData,
+          },
+        })
+      renderWithData(true, {
+        ...claimData,
+      })
+      await waitFor(() => fireEvent.press(screen.getByText('Files')))
+      await waitFor(() => fireEvent.press(screen.getByText('Files')))
```

Comment:
Does this need to be pressed twice?

Reply:
unfortunately it did, just like the details tests above. I haven't figured out why